MinecraftClient/config/ChatBots/discordWebhook.cs

```diff
@@ -0,0 +1,369 @@
+//MCCScript 1.0
+//using System.Collections.Specialized;
+
+MCC.LoadBot(new DiscordWebhook());
+
+//MCCScript Extensions
+class SkinAPI
+{
+    private Dictionary<string, string> skinModes = new Dictionary<string, string>();
+    private string currentSkinMode;
+    private int size;
+    private int scale;
+    private bool overlay;
+    private string fallbackSkin;
+
+    public SkinAPI(string newCurrentSkinMode = "flatFace", string newFallbackSkin = "MHF_Steve", int newSize = 512, int newScale = 10, bool newOverlay = true)
+    {
+        skinModes.Add("flatFace", "https://crafatar.com/avatars/{0}");
+        skinModes.Add("cubeHead", "https://crafatar.com/renders/head/{0}");
+        skinModes.Add("fullSkin", "https://crafatar.com/renders/body/{0}");
+
+        currentSkinMode = newCurrentSkinMode;
+        fallbackSkin = newFallbackSkin;
+        size = newSize;
+        scale = newScale;
+        overlay = newOverlay;
+    }
+
+    // Always get a an Answer to a valid name!
+    public string getUUIDFromMojang(string name)
```

Comment:
Method naming in C# should be `PascalCase`, which the first letter of each word in a compound word is capitalised. The method name would be `GetUUIDFromMojang()`.